modeling/algorithm/local-shared-algorithm_test.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package algorithm
+
+import (
+	"testing"
+
+	"github.com/googleinterns/k8s-topology-simulator/modeling/types"
+)
+
+func TestLocalSharedAlgorithm(t *testing.T) {
+	testCases := []testCase{
+		testCase{
```

Comment:
I don't think you need to specify the type here
```suggestion
		{
```

Reply:
changed